.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Continous Integration
+on: push
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y curl qemu-utils python3-vmdkstream git libxml2-utils libfuse2 nbdkit
+      - name: Cache Turbo
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules/.cache/turbo'
+          key: ${{ runner.os }}-turbo-cache
+          restore-keys: |
+            ${{ runner.os }}-turbo-
```

Comment:
This entry is optional, why do we need it here?

Reply:
At first, my cache key was ```${{ runner.os }}-turbo-${{ github.sha }}``` so I needed this restore-key.

But finally I removed the variable part of the key and I forgot to remove the restore-key as well. I'll do it right now!